all-requirements.txt

```diff
@@ -2,7 +2,7 @@ cliff>=2.0.0,<2.1.0
 eventlet!=0.18.3,!=0.20.1,<0.21.0,>=0.18.2
 gunicorn>=19.4.5
 lxml
-plyvel>=0.9
+plyvel==0.9
```

Comment:
If we know plyvel 1.0.0 is broken, can't we just exclude this version? I'm thinking of something like `plyvel>=0.9,!=1.0.0`.

Reply:
This is only a temporary workaround, likely to be reverted when a proper fix is found...

Reply:
plyvel 1.0.0 requires leveldb 1.20, and our Travis build installs `libleveldb1` 1.15.0 (and there is no superior version in this distribution), thus we are stuck with `plyvel<1.0.0`. Now I'm suggesting we use `plyvel>=0.9,<1.0.0` in case someone creates updates over the 0.9 version.